src/components/Tool/FixSendNewsletter/Contact.tsx

```diff
@@ -197,109 +197,101 @@ const Contact = ({
-              >
-                {contact.primaryPerson.firstName && matches && (
-                  <Avatar
-                    src={contact?.avatar}
-                    style={{
-                      width: theme.spacing(4),
-                      height: theme.spacing(4),
-                    }}
-                  />
-                )}
-                <Box display="flex" flexDirection="column" ml={2}>
-                  <Typography variant="subtitle1">
-                    {`${contact.primaryPerson.firstName} ${contact.primaryPerson.lastName}`}
-                  </Typography>
-                  <Link
-                    underline="hover"
-                    href={`mailto:${contact.primaryPerson.primaryEmailAddress?.email}`}
-                  >
-                    <Typography variant="body2">
-                      {contact.primaryPerson.primaryEmailAddress?.email || ''}
-                    </Typography>
-                  </Link>
-                  {contact.primaryPerson.optoutEnewsletter && (
-                    <Typography variant="body2">
-                      {t('opted out of newsletter')}
+        <Grid container alignItems="center">
+          <Grid item xs={6} sm={5} md={4}>
+            <Box display="flex" alignItems="center" style={{ height: '100%' }}>
+              {contact.primaryPerson?.firstName && matches && (
+                <Avatar
+                  src={contact?.avatar}
+                  style={{
+                    width: theme.spacing(4),
+                    height: theme.spacing(4),
+                  }}
+                />
+              )}
+              <Box display="flex" flexDirection="column" ml={2}>
+                {contact.primaryPerson?.firstName &&
+                  contact.primaryPerson?.lastName && (
+                    <Typography variant="subtitle1">
+                      {`${contact.primaryPerson?.firstName} ${contact.primaryPerson?.lastName}`}
                     </Typography>
                   )}
-                </Box>
-              </Box>
-            </Grid>
-            <Grid item xs={6} sm={4} md={4}>
-              <Box
-                display="flex"
-                alignItems="start"
-                flexDirection="column"
-                p={2}
-              >
-                <Typography variant="body2">
-                  {contact?.primaryAddress?.street || ''}
-                </Typography>
-                <Typography variant="body2">
-                  {`${contact.primaryAddress?.city || ''} ${
-                    contact.primaryAddress?.state
-                      ? contact.primaryAddress.state
-                      : ''
-                  } ${contact.primaryAddress?.postalCode || ''}`}
-                </Typography>
-                <Typography variant="body2">
-                  {contact.primaryAddress?.country || ''}
-                </Typography>
-                {contact.primaryAddress?.source && (
+                <Link
```

Comment:
It seems like we might want to hide this link if the user doesn't have a primary email address? I guess an empty link won't render anything, but it still seems a little more correct to render nothing.